src/Components/NavBar/NavBar.js

```diff
@@ -27,23 +27,24 @@ const NavBarTopStyled = styled.div`
   justify-content: space-between;
   font-family: "News Cycle", sans-serif;
   color: white;
-`;
-const DropDownStyled = styled.select`
-  background-color: black;
-  width: 15%;
-  height: 40%;
-  border-radius: 5px;
-  color: white;
-  margin: 3%;
-  font-size: 1em;
-  border: 1px solid white;
-`;
-const SignInBtnStyled = styled.button`
-  border: 2px solid black;
-  border-radius: 5px;
-  background-color: white;
-  margin: 3%;
-`;
+
+  h1 {
+    margin: 3%;
+    padding: 1.5%;
+    border: 1px solid white;
+  }
+
+  button {
+    border: 2px solid black;
+    border-radius: 5px;
+    background-color: white;
+    margin: 3%;
+  }
+
+  h3 {
+    text-decoration: underline
+  }
+`
 
 const NavBarBottomStyled = styled.div`
   width: 100%;
```

Comment:
I think we might be able to refactor further to have just one NavBarStyled component but looks good. Thanks!